### PR TITLE
switch to d2l-test-runner

### PR DESCRIPTION
**Pull request by dlockhart**
I wanted to remove the [keyDown helper](https://github.com/BrightspaceUI/core/blob/main/tools/dom-test-helpers.js#L3) alongside `getUpdateCompleteAll`, and this was the only place using it. So converting this to `d2l-test-runner` it is!

### CODE REVIEW

File: d2l-focusable-arrowkeys-behavior.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ D2L.PolymerBehaviors.FocusableArrowKeysBehavior = {
 		var target = e.target;
 		if (this.arrowKeyFocusablesDirection.indexOf('left') >= 0 && e.keyCode === this.__keyCodes.LEFT) {
 			fastdom.measure(function() {
+				if (!this.isConnected) return;
```
Comment (dlockhart) [inline d2l-focusable-arrowkeys-behavior.js:131]: There were some weird timing-related errors due to the async nature of this code and it running _after_ the tests had moved on and the nodes no longer being connected anymore.

File: test/focusable-arrowkeys-behavior.test.js [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
 				};
-				focusables[keyInteraction.startIndex].focus();
-				focusables[keyInteraction.endIndex].addEventListener('focus', () => {
-					expect(D2L.Dom.Focus.getComposedActiveElement()).to.equal(focusables[keyInteraction.endIndex]);
-					done();
-				});
-				keyDown(focusables[keyInteraction.startIndex], keyInteraction.keyCode);
+				const e = oneEvent(focusables[keyInteraction.endIndex], 'focus');
+				sendKeysElem(focusables[keyInteraction.startIndex], 'press', keyInteraction.key);
+				await e;
```
Comment (svanherk) [inline test/focusable-arrowkeys-behavior.test.js]: Was this happening too fast even without the `await` for this to work?
```
sendKeysElem(focusables[keyInteraction.startIndex], 'press', keyInteraction.key);
await oneEvent(focusables[keyInteraction.endIndex], 'focus');
```

Comment (dlockhart) [reply test/focusable-arrowkeys-behavior.test.js]: Hmm I definitely did try that, but some other stuff may have been broken at the time. Just tried it again and yes that works -- updated!

**Review by svanherk — APPROVED** [review]
Nice!

**Pull request merged**